packages/web-components/src/components/pricing-table/pricing-table-cell.ts

```diff
@@ -26,6 +26,9 @@ const { stablePrefix: ddsPrefix } = ddsSettings;
 class DDSPricingTableCell extends StableSelectorMixin(
   HostListenerMixin(DDSStructuredListCell)
 ) {
+  @property({ type: Boolean, attribute: 'icon-text', reflect: true })
+  iconText?: boolean = false;
```

Comment:
You don't need the `?:` (what is this called??) here since you're setting a default value.
```suggestion
  iconText: boolean = false;
```

Reply:
"[optional value](https://www.typescripttutorial.net/typescript-tutorial/typescript-optional-parameters/)"

Reply:
I'm not sure how I feel about using a new parameter for this - would it be better to just assume that text & icon use together warrants displaying both?

Reply:
I agree that would be way better. The thing is, when I went looking for examples that already exist in the wild, they all seem to have some default text "Cell with icon" that we wouldn't want suddenly appearing :(

https://www.ibm.com/products/instana/pricing
https://www.ibm.com/products/qradar-edr/pricing
https://www.ibm.com/products/planning-analytics/pricing

I suppose if they are adding that default text in a template, we could co-ordinate them removing that when this update gets adopted, but I'm not sure how realistic that would be. That's why I did this as an opt-in.

Open to changing it though, what do you think?

Reply:
I asked Kayle and Antonio in a comment on the ticket if removing the default text is possible.

I hope it is, because I have fairly strong opinions about avoiding workarounds for AEM's implementation details. Those can always change, and our code could be left with features/behaviors that are unclear to future maintainers or even ourselves in a few months.

What do we think about documenting the decision in a comment should we need to leave this as opt-in?

Reply:
> I asked Kayle and Antonio in a comment on the ticket if removing the default text is possible.
> I hope it is, because I have fairly strong opinions about avoiding workarounds for AEM's implementation details.

Sounds good. Will wait on what they come back with. 🤞 

>  What do we think about documenting the decision in a comment should we need to leave this as opt-in?

Yep, can do.

Reply:
@m4olivei Good news! The AEM team can remove the default text from existing content, so we're able to avoid the opt-in approach here.

Reply:
Excellent! I'll update things here accordingly.